timemachine/fe/free_energy.py

```diff
@@ -37,6 +38,16 @@ class MDParams:
     n_frames: int
     n_eq_steps: int
     steps_per_frame: int
+    local_steps: int = 0
+    k: float = 10_000.0
```

Comment:
mild surprise: thought the filtering suggested smaller values of k were also acceptable, and we used k = 1_000.0 by default in most of the figures

Reply:
Yes, this was just to match the default value of `multiple_steps_local`, will change it to `k=1000.0`.

Reply:
Changed to 1k in https://github.com/proteneer/timemachine/pull/1026/commits/29680b9a2d6ff6c72d388b85c38a5da892b69161